apps/client/src/components/Location/Location.js

```diff
@@ -66,16 +70,79 @@ const LocationPage = ({ topic }) => {
       if (focus) {
         document.activeElement.blur();
       } else {
-        history.push(geturl(routes.address, topic));
+        setAddressShown(true);
+        if (checkerContext.checker) {
+          checkerContext.checker.next();
+        }
       }
     }
   };
+  const getOloUrl = ({ postalCode, houseNumberFull, houseNumber }) => {
```

Comment:
Are we going to split OLO flow from the STTR flow?

Reply:
I think this needs discussion with Annick and Robin.

Reply:
We're going to try to implement the OLO flow in the one pager.